nltools/data/brain_data.py

```diff
@@ -83,11 +86,17 @@
 from scipy.signal import detrend
 from scipy.spatial.distance import squareform
 import six
-from sklearn.metrics.pairwise import pairwise_distances
+from sklearn.metrics.pairwise import (pairwise_distances,
```

Comment:
Not sure what happened here, but we only need pairwise_distances from sklearn.  

transform_pairwise, and summarize_bootstrap should be in nltools.stats module

Reply:
yea I fixed this already. GH is just showing history cause it's broken up into a bunch of commits on this PR.